assets/js/components/FeatureTours.js

```diff
@@ -65,7 +69,7 @@ export default function FeatureTours() {
 		return () => {
 			observer.disconnect();
 		};
-	}, [ tour ] );
```

Comment:
I didn't spot this earlier, looks like this dependency array could have been removed by accident or was it deliberate?

Reply:
Seems like a mistake – I'd expect such an exception to be annotated if it were indeed necessary for some reason.

Reply:
Oh, yup, totally a mistake from testing. I've added it back! 👍🏻